components/brave_wallet/browser/ethereum_provider_impl.cc

```diff
@@ -502,7 +502,8 @@ void EthereumProviderImpl::RecoverAddress(const std::string& message,
   }
 
   reject = false;
-  std::move(callback).Run(std::move(id), base::Value(address), reject, "",
+  std::move(callback).Run(std::move(id),
+                          base::Value(std::move(address).value()), reject, "",
```

Comment:
Let's have it as `std::move(address.value())`

Reply:
I'm gonna leave it as I don't want to incur on another CI run over something that is immaterial.